tests/phpunit/api/v3/ActivityTest.php

```diff
@@ -447,6 +447,11 @@ public function testActivityCreateExample() {
     require_once 'api/v3/examples/Activity/Create.php';
     $result = activity_create_example();
     $expectedResult = activity_create_expectedresult();
+    // Mask timestamp fields. Too variable.
```

Comment:
Sorry, but I'm afraid I don't understand this comment. Is is because the timestamps will change every time you run the test?